JSON_for_IO-Link.yaml

```diff
@@ -3596,8 +3596,8 @@ paths:
       operationId: GetDevicesDeviceAliasFirmwareUpdateStatus
       tags:
         - devices
-      description: Gives the stage and progress of a running firmware update. The active property shows if an update is currently in progress.
-      summary: Gives the stage and progress of a running firmware update. The active property shows if an update is currently in progress.
+      description: Gives the stage and progress of a running firmware update. The active property shows if an update is currently in progress. Device FW Update support is required.
```

Comment:
Change the description like this:
"This endpoint is mandatory if the master supports device FW updates."